prime-angular-frontend/src/app/modules/enrolment/pages/access-declined/access-declined.component.html

```diff
@@ -0,0 +1,25 @@
+<app-enrollee-page [busy]="busy">
+
+  <app-page-header>PharmaNet Enrolment</app-page-header>
+
+  <app-enrolment-progress-indicator *ngIf="isInitialEnrolment"
+                                    message="Enrolment Declined"
+                                    [currentRoute]="EnrolmentRoutes.ACCESS_DECLINED"></app-enrolment-progress-indicator>
+
+  <div class="mb-5">
+    <app-alert type="danger">
+      <ng-container #alertContent
```

Comment:
Can you update the alert to have a title like the access-locked page "Your PRIME enrolment has been declined", and it the content of the alert looks like it should be "Your application for PharmaNet access has not be approved."